profile.md

```diff
@@ -232,6 +229,37 @@ part of a Kubernetes API call:
   }
   ```
 
+## Bind Resource Object
+
+In the [Open Service Broker API specification](spec.md), requests to
+[create a Service Binding](spec.md#binding) can contain a `bind_resource`
+object in which Platforms MAY choose to add additional fields.
+
+#### Cloud Foundry Bind Resource Object
+
+The following properties are defined for usage within a Cloud Foundry
+deployment:
+
+- `space_guid` (_optional_)  
+  The GUID of a space that a Service Binding is associated with. If present,
+  this property MUST be a non-empty string serialized as follows:
+  ```
+  "space_guid": "space-guid-here"
+  ```
+  For example:
+  ```
+  "space_guid": "15823972-c216-4ba5-9f3f-e75b0b891297"
+  ```
+
+The following example shows a `bind_resource` object that might appear as part
```

Comment:
When "might" it appear?

Reply:
When a CF deployment supports it, and when a binding is being created to an application

Reply:
I would suggest you replace might, with the exact example / CF deployment version supported from.

Is there any way for a client to know when to depend on it from the API header?

Reply:
Currently we just version this file, so broker authors will know it is supported when CF sends `2.14` as `x-broker-api-version`. That's probably enough right?